pkg/simulation/simulation.go

```diff
@@ -53,6 +54,14 @@ type simulation struct {
 	actionTargets map[key.TargetID]bool
 }
 
+func RunWithLog(cfg *model.SimConfig, eval *eval.Eval, seed int64) (*model.IterationResult, error) {
+	cfg.Iterations = 1
```

Comment:
Should not modify cfg/treat it as read-only. simulation always runs a single iteration, so don't have enough context to set it to 1 here